backends/cuda/ceed-cuda-basis.c

```diff
@@ -415,6 +500,67 @@ int CeedBasisApply_Cuda(CeedBasis basis, const CeedInt nelem,
   return 0;
 }
 
+int CeedBasisApplyNonTensor_Cuda(CeedBasis basis, const CeedInt nelem,
```

Comment:
It looks like this will need the adjustment to remove "ceed-impl.h" as well.

Reply:
That's right, but most of these are already in cuda-shared branch, I'll rebase once the shared-memory backend gets merged into master.